src/irc/ClientPool.ts

```diff
@@ -403,6 +403,7 @@ export class ClientPool {
         }
 
         if (bridgedClient.explicitDisconnect) {
+            log.info(`Dropping ${bridgedClient.id} (${bridgedClient.nick}) because explicitDisconnect is true`);
```

Comment:
I know the guy in the issue said his log level was only `warn`, but this sounds like a `debug` log line more than an `info` one.

Reply:
Debatable, we wouldn't expect to see a ton of these because not many people disconnect on purpose. I would argue the information is rather vital?